src/app/client/src/app/modules/public/components/public-collection-player/public-collection-player.component.ts

```diff
@@ -153,10 +154,16 @@ export class PublicCollectionPlayerComponent implements OnInit, OnDestroy {
     });
   }
 
-  public OnPlayContent(content: { title: string, id: string }) {
+  public OnPlayContent(content: { title: string, id: string }, isCalledFromOnPlay) {
```

Comment:
the variable name should be more sensible

Reply:
done